eeps/eep-0015.md

```diff
@@ -65,35 +61,37 @@ a parameter.
 The built-in function erlang:fun_info/1 is extended in
 the following ways:
 
-1.  In a {type,Type} item, Type may be 'portable'.
-2.  In a {module,Module} item for a portable fun, the Module
-    will be present, but there will in fact be no other
-    connection between a portable fun and any module by that name.
-3.  In a {name,Name} item for a portable fun,
-    Name will always be [].
-4.  None of the items specified for 'local' funs will be
-    returned for 'portable' funs.
-5.  {calls,List} will be returned for a portable fun,
-    where List is a list of {Module,Imports} pairs, where
-    each Module that is used in a remote call in the fun is
-    listed once, and the Imports are a list of {Name,Arity}
-    pairs as reported in *:module_info/0.  This permits the
-    receiver of a portable fun to determine which modules
-    need loading and which functions they are expected to export.
-6.  For consistency,
-    erlang:fun_info(fun M:F/A, calls)
-    => [{M,[{F,A}]}]
+1. In a {type,Type} item, Type may be 'portable'.
+2. In a {module,Module} item for a portable fun, the Module
+   will be present, but there will in fact be no other
+   connection between a portable fun and any module by that name.
+3. In a {name,Name} item for a portable fun,
+   Name will always be [].
+4. None of the items specified for 'local' funs will be
+   returned for 'portable' funs.
+5. {calls,List} will be returned for a portable fun,
+   where List is a list of {Module,Imports} pairs, where
+   each Module that is used in a remote call in the fun is
+   listed once, and the Imports are a list of {Name,Arity}
+   pairs as reported in *:module_info/0.  This permits the
+   receiver of a portable fun to determine which modules
+   need loading and which functions they are expected to export.
+6. For consistency,
+   erlang:fun_info(fun M:F/A, calls)
+   => [{M,[{F,A}]}]
 
 The built-in function erlang:fun_info/2 is extended similarly.
 An additional key 'source' is provided for this function.
 
-###fun_info(Fun, source)###
--   for a local fun, the result is 'undefined'.
--   for an external fun, the result is the abstract syntax
-    tree the parser returns for fun M:F/A.
--   for a portable fun, the result is the abstract syntax
-    tree the parser returned for the fun!..... end form
-    it came from.
+fun_info(Fun, source)
```

Comment:
I'm not sure this was intended as a title, since it's not formatted as one, so I tried to guess, here.